bin/alluxio

```diff
@@ -181,7 +181,7 @@ function main {
   if [[ $- == *x* ]]; then
     LAUNCHER="bash -x"
   fi
-  BIN=$(cd "$( dirname "$( readlink "$0" )" )"; pwd)
+  BIN=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
```

Comment:
We'd need to put that function in a common file, but then we can't find the file without calling the function first x.x